docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,174 @@
+.. _Courses API Courses Resource:
+
+########################################
+Courses API Courses Resource
```

Comment:
OK, I'm confused. A conversation with @clintonb and @nasthagiri yesterday led me to understand that the Course Catalog API consumes the Courses API here (with "Course Catalog API" being the API that ECOM has been working on and "Courses API" being the API that Nimisha's team has been working on). Is this not the case?

@pdesjardins @catong  FYI.